### ISSUE TITLE
1.4.19-SNAPSHOT conflicts with Paper 1.16.5

### ISSUE THREAD

**Issue by Dymeth** [InventivetalentDev/GlowAPI#90]
Hello!
An exception is thrown several times per second: 

```
[PacketListenerAPI] An exception occured while trying to execute 'onSend' in plugin GlowAPI: java.lang.ClassNotFoundException: Could not resolve class for [ResolverQuery{name='net.minecraft.server.v1_16_R3.LevelEntityGetter', types=[]}]
java.lang.RuntimeException: java.lang.ClassNotFoundException: Could not resolve class for [ResolverQuery{name='net.minecraft.server.v1_16_R3.LevelEntityGetter', types=[]}]
	at org.inventivetalent.glow.GlowAPI.getEntityById(GlowAPI.java:701)
	at org.inventivetalent.glow.GlowAPI$1.onSend(GlowAPI.java:588)
	at org.inventivetalent.packetlistener.handler.PacketHandler.notifyHandlers(PacketHandler.java:81)
	at org.inventivetalent.packetlistener.PacketListenerAPI.onPacketSend(PacketListenerAPI.java:116)
	at org.inventivetalent.packetlistener.channel.ChannelAbstract.onPacketSend(ChannelAbstract.java:93)
	at org.inventivetalent.packetlistener.channel.INCChannel$ChannelHandler.write(INCChannel.java:128)
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717)
	at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:764)
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:790)
	at io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:758)
	at io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:808)
	at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:1025)
	at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:294)
	at com.comphenix.protocol.injector.netty.ChannelProxy.writeAndFlush(ChannelProxy.java:327)
	at net.minecraft.server.v1_16_R3.NetworkManager.lambda$b$6(NetworkManager.java:323)
	at com.comphenix.protocol.injector.netty.ChannelInjector$3.lambda$onMessageScheduled$1(ChannelInjector.java:351)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:472)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at java.base/java.lang.Thread.run(Thread.java:831)
Caused by: java.lang.ClassNotFoundException: Could not resolve class for [ResolverQuery{name='net.minecraft.server.v1_16_R3.LevelEntityGetter', types=[]}]
	at org.inventivetalent.glow.reflection.resolver.ClassResolver.notFoundException(ClassResolver.java:42)
	at org.inventivetalent.glow.reflection.resolver.ClassResolver.notFoundException(ClassResolver.java:9)
	at org.inventivetalent.glow.reflection.resolver.ResolverAbstract.resolve(ResolverAbstract.java:63)
	at org.inventivetalent.glow.reflection.resolver.ClassResolver.resolve(ClassResolver.java:29)
	at org.inventivetalent.glow.reflection.resolver.minecraft.NMSClassResolver.resolve(NMSClassResolver.java:28)
	at org.inventivetalent.glow.GlowAPI.getEntityById(GlowAPI.java:665)
	... 21 more
```

Configuration:
git-Paper-778 (MC: 1.16.5) (Implementing API version 1.16.5-R0.1-SNAPSHOT)
PacketListenerApi version 3.9.3-SNAPSHOT
GlowAPI version 1.4.19-SNAPSHOT
[ReflectionHelper/MinecraftVersion] Version is v1_16_R3 (11603)

**Comment by Dymeth** [comment]
Not fully resolved on 1.4.19-SNAPSHOT.
The net.minecraft.server.v1_16_R3.LevelEntityGetter class is still used, which is missing as of 1.16.5

**Comment by Dymeth** [comment]
Solved in 1.4.20-SNAPSHOT, thank you <3
